INSTALL.md

```diff
@@ -93,9 +93,11 @@ These steps place the compiled KeePassXC binary inside the `./build/src/` direct
 
 	```
 	  -DWITH_XC_AUTOTYPE=[ON|OFF] Enable/Disable Auto-Type (default: ON)
-	  -DWITH_XC_HTTP=[ON|OFF] Enable/Disable KeePassHTTP and custom icon downloads (default: OFF)
 	  -DWITH_XC_YUBIKEY=[ON|OFF] Enable/Disable YubiKey HMAC-SHA1 authentication support (default: OFF)
 	  -DWITH_XC_BROWSER=[ON|OFF] Enable/Disable KeePassXC-Browser extension support (default: OFF)
+      -DWITH_XC_NETWORKING=[ON|OFF] Enable/Disable Networking support (favicon download) (default: OFF)
+
+      -DWITH_XC_ALL=[ON|OFF] Enable/Disable compiling all plugins above (default: OFF)
```

Comment:
This seems to have different spacing from the rest of the stuff here – looks like they use a combination of spaces and tabs, whereas this is just spaces?